templates/shortcode-form-grid.php

```diff
@@ -293,7 +293,7 @@ static function ($term) use ($tag_text_color, $tag_bg_color) {
                     $style .= "background: linear-gradient(180deg, {$color} 0%, {$color} 100%); background-blend-mode: multiply;";
                     echo "
                                             <div class='give-form-grid-progress-bar'>
-                                                    <div class='give-progress-bar' role='progressbar' aria-valuemin='0' aria-valuemax='100' aria-valuenow='$progress_bar_value'>
+                                                    <div class='give-progress-bar' role='progressbar' aria-valuemin='0' aria-valuemax='100' aria-valuenow='$progress_bar_value' aria-label='Form progress: $progress_bar_value% toward goal'>
```

Comment:
Let's make this string translatable, also I wonder if we have to spell out the word "percent" rather than use the percent sign (i'm not sure)

Reply:
Resolved: 23ad87838015b4141124d39c0e420f05121177f0

Heres how the built in mac tool reads it.

https://github.com/user-attachments/assets/e51f5f2c-8af5-4caa-80d0-8850467576e5